<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCompilerClass</class>
 <widget class="QMainWindow" name="CCompilerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CCompiler</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="PathLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>741</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>请选择目标文件路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectFileButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="CodeTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>521</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>20</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始分析</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ResultTextEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>521</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>源代码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>分析结果</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CCompiler.qrc"/>
 </resources>
 <connections/>
</ui>
